src/OpenLoco/src/Objects/ScaffoldingObject.h

```diff
@@ -37,38 +37,38 @@ namespace OpenLoco
 
     namespace Scaffolding::ImageIds
     {
-        constexpr uint32_t type01x1SegmentBack = 0;
-        constexpr uint32_t type01x1SegmentFront = 1;
+        constexpr uint32_t type01x1SegmentFlagsBack = 0;
+        constexpr uint32_t type01x1SegmentFlagsFront = 1;
```

Comment:
None of these are flags. I think these changes should be reverted. @duncanspumpkin, thoughts?

Reply:
yeah that looks like it was a whoopsie